### DISCUSSION TITLE
Error when loading   model: Model STT En Conformer-CTC Large was not found

### DISCUSSION THREAD

**Discussion by vitotitto** [NVIDIA-NeMo/NeMo#3945] [Q&A] [answered]
I'm trying to load the model "STT En Conformer-CTC Large" with the line : 
`nemo_asr.models.ASRModel.from_pretrained(model_name="stt_en_conformer_ctc_large")` 

However I get the error: FileNotFoundError: Model stt_en_conformer_ctc_large was not found. Check cls.list_available_models() for the list of all available models. 

What is the correct way of doing this?

**Comment by titu1994** [comment, marked as answer]
What Nemo version are you currently using ? Please upgrade to 1.7.2. 

If that doesn't work then use EncDecCTCModelBPE.from_pretrained() instead of ASRModel

**Reply by vitotitto** [reply]
Thank you so much!  It works!
